### PR TITLE
add netdata_stream resource

### PR DESCRIPTION
**Pull request by salsa-dev**
inspec only for the slave NetData

### CODE REVIEW

**Comment by salsa-dev** [comment]
for ruby 2.4.0 it is ok, ruby 2.2.2 fails
```
/opt/chefdk/embedded/bin/ruby -I/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-core-3.7.0/lib:/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-support-3.7.0/lib /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-core-3.7.0/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb failed
serge@ci:~/DEV/netdata-cookbook$ /opt/chefdk/embedded/bin/ruby -I/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-core-3.7.0/lib:/opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-support-3.7.0/lib /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/rspec-core-3.7.0/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb


Randomized with seed 30222

netdata_test::default
  bind_rndc custom configuration
    uses netdata_python_plugin as the backend for bind_rndc
    configures bind_rndc
    creates bind_rndc configuration file with custom path
    creates bind_rndc python.d plugin configuration file
    converges successfully

netdata::install_netdata
  Ubuntu
    includes apt cookbook
    installs compile packages
    installs the python mysql package for NetData mysql plugin
    behaves like install
      restarts NetData service on install
      enables and starts the NetData service
      runs the netdata install custom resource
      checks out NetData git repository
      installs the NetData service
      installs packages for NetData plugins
      converges successfully

netdata::default
  When all attributes are default
    converges successfully
    includes install_netdata recipe

netdata::install_netdata
  CentOS
    includes yum-epel cookbook
    installs compile packages
    behaves like mysql_plugin
      installs the python mysql package for NetData mysql plugin
    behaves like install
      runs the netdata install custom resource
      installs packages for NetData plugins
      installs the NetData service
      restarts NetData service on install
      checks out NetData git repository
      enables and starts the NetData service
      converges successfully

netdata_test::default
  netdata_python_plugin custom mysql configuration
    configures netdata_python_plugin
    creates mysql configuration file with custom jobs
    creates mysql python.d plugin configuration file
    creates mysql configuration file with custom global configuration
    converges successfully

netdata_test::default
  nginx_conf custom configuration
    creates nginx configuration file with custom path
    converges successfully
    creates nginx python.d plugin configuration file
    uses netdata_python_plugin as the backend for nginx
    configures nginx_conf

netdata_test::default
  netdata_config custom configuration
    configures netdata_config subsection: global
    configures netdata_config subsection: plugin:proc:/proc/meminfo
    converges successfully
    configures netdata_config subsection: web

netdata::install_netdata
  Fedora
    installs compile packages
    behaves like mysql_plugin
      installs the python mysql package for NetData mysql plugin
    behaves like install
      checks out NetData git repository
      restarts NetData service on install
      runs the netdata install custom resource
      converges successfully
      installs packages for NetData plugins
      enables and starts the NetData service
      installs the NetData service

netdata_test::default
  netdata_stream custom configuration
    configures netdata_stream subsection: stream
    configures netdata_stream subsection: api_key
    converges successfully

Finished in 1.76 seconds (files took 3.61 seconds to load)
53 examples, 0 failures

Randomized with seed 30222


ChefSpec Coverage report generated...

  Total Resources:   26
  Touched Resources: 23
  Touch Coverage:    88.46%

Untouched Resources:

  apt_package[compile_packages]      netdata/resources/netdata_install.rb:56
  yum_package[compile_packages]      netdata/resources/netdata_install.rb:56
  dnf_package[compile_packages]      netdata/resources/netdata_install.rb:56


```

**Comment by salsa-dev** [comment]
PR is ready for approval/merge. Looking forward to your feedback.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,42 @@ Resulting file content (/etc/netdata/netdata.conf):
...
+Manages stream.conf file. Call this as many times as needed.  
+Resource names could be either 'stream' or "#{api_key}" and "#{machine_guid}" depending on whether you configure slave or master NetData.
+Name 'stream' should be used only once to configure slave NetData.
+Values for api_key and machine_guid should be unique.
+
+```rb
+netdata_stream 'stream' do
+  owner 'netdata'
+  group 'netdata'
+  base_directory "#{instal_path}/netdata"
```
Comment (nictrix) [inline README.md]: missing an `l`

**Comment by salsa-dev** [comment]
fixed. thanks

**Comment by jmadureira** [comment]
Thank you @salsa-dev :)

**Pull request merged**